packages/teams-js/src/internal/communication.ts

```diff
@@ -264,33 +269,33 @@ export function requestPortFromParentWithVersion(
     );
   }
   const request = sendMessageToParentHelper(apiVersionTag, actionName, args);
-  return waitForPort(request.id);
+  return waitForPort(request.uuid);
 }
 
 /**
  * @internal
  * Limited to Microsoft-internal use
  */
-function waitForPort(requestId: number): Promise<MessagePort> {
+function waitForPort(requestId: MessageUUID): Promise<MessagePort> {
   return new Promise<MessagePort>((resolve, reject) => {
-    CommunicationPrivate.portCallbacks[requestId] = (port: MessagePort | undefined, args?: unknown[]) => {
+    CommunicationPrivate.portCallbacks.set(requestId, (port: MessagePort | undefined, args?: unknown[]) => {
       if (port instanceof MessagePort) {
         resolve(port);
       } else {
         // First arg is the error message, if present
         reject(args && args.length > 0 ? args[0] : new Error('Host responded without port or error details.'));
       }
-    };
+    });
   });
 }
 
 /**
  * @internal
  * Limited to Microsoft-internal use
  */
-function waitForResponse<T>(requestId: number): Promise<T> {
+function waitForResponse<T>(requestId: MessageUUID): Promise<T> {
```

Comment:
```suggestion
function waitForResponse<T>(requestuuId: MessageUUID): Promise<T> {
```
because the old number value is still called messageId I had to think for a moment to figure out if this was correct. A namechange away from id to uuid might help?

Reply:
good catch, I meant to do this